.github/workflows/ci.yml

```diff
@@ -193,11 +193,14 @@ jobs:
           fi
           echo "PHP has been configured"
 
-      - name: Apt install additional requirements 
+      - name: Install additional requirements 
         env:
           GITHUB_REPOSITORY: ${{ github.repository }}
+          # These are used for aws-cli, which is pre-installed in github actions. This is only for dynamodb
```

Comment:
Does this mean these are required for dynamodb installation to work?

Reply:
Actually we can probably get rid of these now that we're not doing the `aws dynamodb` commands in bash anymore